### ISSUE TITLE
PR this to sunkit-image

### ISSUE THREAD

**Issue by wtbarnes** [Cadair/sunspot_experiments#2]
We should try to get this into sunkit-image as it is a generally useful algorithm. Is there anything blocking this?

**Comment by Cadair** [comment]
No, I don't think so, we should do that. There is an issue on sunkit-image for this algorithm.
